frontend/src/components/FavoriteGallery.tsx

```diff
@@ -0,0 +1,21 @@
+import {FavoriteNft} from "../model/FavoriteNft";
+import FavoriteCard from "./FavoriteCard";
+
+
+
+type FavoriteGalleryProps = {
+    favoriteNfts:FavoriteNft[];
+}
+
+export default  function FavoriteGallery(props:FavoriteGalleryProps){
+
+    return (
+     <div className ="favoritecard">
+         {props.favoriteNfts.map((FavoriteNft) =>
```

Comment:
FavoriteNft should be {props.favoriteNfts.map((favoriteNft) =>
I am not sure if you are using your props correctly.